mozilla/fx-private-relay
# Mask-specific counts of blocked trackers

# New feature description

This adds a count to individual masks to display how many trackers have been blocked for emails sent to that mask.

Note that the back-end does not yet provide this data; the UI will not be visible until it does. Use the mock API to see the UI today.

Note that I've attempted to split out the tracker blocking work in as small pieces as possible, though there's still some overlap in the different PRs. If it looks like things are missing, the complete work is available in the branch [`trackerblocking-all`](https://github.com/mozilla/fx-private-relay/tree/trackerblocking-all).

# Screenshot (if applicable)

![](https://user-images.githubusercontent.com/4251/174631674-6f9b82f4-00dd-4390-a7b2-6ecc548cde39.png)

![image](https://user-images.githubusercontent.com/4251/174638193-28a16d12-4993-485d-94d0-7231c172ab32.png)

# How to test

Open [the mock site with the user `full`](https://deploy-preview-2100--fx-relay-demo.netlify.app/?mockId=full). There should be a count next to every mask that shows a tooltip on hover.

Also note that this implements the mobile view for mask-specific stats.

# Checklist

- [ ] l10n dependencies have been merged, if any. - Submitted here: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/91
